app/src/main/java/com/example/poste/utils/ValidationUtils.java

```diff
@@ -76,10 +90,14 @@ public static boolean validatePassword(Context context, String password, String
      */
     public static boolean validatePassword(Context context, String password) {
         if (password.length() < 8) {
-            Toast.makeText(context, "Password must be at least 8 characters!", Toast.LENGTH_SHORT).show();
+            Toast.makeText(context,
+                    R.string.password_short,
```

Comment:
Suggest name like "error_password_short" so it is clear this is an error message of some type (as opposed to just a length)